CHANGELOG.next.asciidoc

```diff
@@ -111,6 +111,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Set timeout of 1 minute for FQDN requests {pull}37756[37756]
 - Fix issue where old data could be saved in the memory queue after acknowledgment, increasing memory use {pull}41356[41356]
 - Ensure Elasticsearch output can always recover from network errors {pull}40794[40794]
+- Add `translate_guid` processor for windows platforms. {pull}41472[41472]
```

Comment:
Shouldn't this processor include `ldap` somewhere in its name, for clarity?

Reply:
Agreed, initally was expected to be less generic.